doc/sphinx/conf.py

```diff
@@ -220,6 +220,7 @@ def setup(app):
     'versions': [
         ("dev", "https://coq.github.io/doc/master/refman/"),
         ("stable", "https://coq.inria.fr/distrib/current/refman/"),
+        ("8.16", "https://coq.github.io/doc/v8.16/refman/"),
         ("8.15", "https://coq.github.io/doc/v8.15/refman/"),
```

Comment:
Could you also turn this line into the following? This will be broken for a while but should be fixed soon enough.
```suggestion
        ("8.15", "https://coq.github.io/doc/V8.15.2/refman/"),
```

Reply:
Done.